codespell_lib/data/dictionary.txt

```diff
@@ -23751,6 +23756,7 @@ segmenetd->segmented
 segmeneted->segmented
 segmenets->segments
 segmentaion->segmentation
+segmente->segment
```

Comment:
Perhaps add the plural version of this too, i.e. tes->ts

Reply:
added